content/general/functions.textile

```diff
@@ -137,7 +122,7 @@ Each member of @presence@ has the following fields:
 
 h4. Decoding presence events
 
-Each member of @presence@ is a 'raw' presence event. Presence messages sent "over the realtime service":/realtime/channels-messages are automatically decoded into "@PresenceMessages@":/realtime/types#presence-message object by the Ably client library. With Reactor Function you need to to do this explicitly, using "@PresenceMessage.fromEncodedArray@":/realtime/presence#presence-from-encoded-array on the @presence@ array, or "@PresenceMessage.fromEncoded@":/realtime/presence#presence-from-encoded on an individual member of that array. This will transform them into an array of "@PresenceMessages@":/realtime/types#presence-message (or in the case of @fromEncoded@, an individual "@PresenceMessage@":/realtime/types#presence-message). This has several advantages, e.g.:
+Each member of @presence@ is a 'raw' presence event. Presence messages sent "over the realtime service":/realtime/channels-messages are automatically decoded into "@PresenceMessages@":/realtime/types#presence-message object by the Ably client library. With Reactor Function you need to to do this explicitly, using "@PresenceMessage.fromEncodedArray@":/realtime/presence#presence-from-encoded-array on the @presence@ array, or "@PresenceMessage.fromEncoded@":/realtime/presence#presence-from-encoded on an individual member of that array. This will transform them into an array of "@PresenceMessages@":/realtime/types#presence-message (or in the case of @fromEncoded@, an individual "@PresenceMessage@":/realtime/types#presence-message). Using the Ably client library to decode @presencemessages@ has several advantages:
```

Comment:
@PresenceMessage@, not @PresenceMessages@.  If you need to refer to plural, as Textile is shit if you do @PresenceMessage@s (it doesn't render propertly), you have to do @PresenceMessage@ objects

Reply:
You also have @presencemessages@?